### DISCUSSION TITLE
Administration Settings->Nextcloud Office page doesn't show WOPI field and doesn't let me change values.

### DISCUSSION THREAD

**Discussion by jhoski** [nextcloud/all-in-one#4455] [Questions] [answered]
I am running all-in-one on debian 12 behind a cloudflare tunnel and it is working except for Office.
The documentation says that in order to get Office to work with cloudflare you need to add cloudflare's ip list to the WOPI list @ 
https://yourdomain.com/settings/admin/richdocuments, however that page does not show the field nor does it it allow me to change any of the values on the page.

Please advise how to fix.

**Comment by szaimen** [comment, marked as answer]
Hi, can you follow https://github.com/nextcloud/all-in-one/discussions/1358?
